libs/common-utils/src/environments.test.ts

```diff
@@ -20,7 +20,7 @@ describe('Detect environments using host and path', () => {
     })
   })
 
-  describe('Is production', () => {
+  describe.only('Is production', () => {
```

Comment:
Ops, should revert this

Reply:
https://github.com/cowprotocol/cowswap/pull/3309